### PR TITLE
Fixes #6503: Update command on centos is missing \"ncf-api-virtualenv\" package

### PR DESCRIPTION
**Pull request by VinceMacBuche**
http://www.rudder-project.org/redmine/issues/6503

### CODE REVIEW

File: 12_upgrade/20_upgrade_centos_rhel.txt [hunk 1]
```diff
@@ -32,7 +32,7 @@ For Rudder Server (RHEL 6 only), upgrade the +rudder-*+ and +ncf+ packages:
 
 ----
 
-yum update "rudder-*" "ncf"
+yum update "rudder-*" "ncf-*"
```
Comment (Kegeruneku) [inline 12_upgrade/20_upgrade_centos_rhel.txt]: No, if you do this you will not update ncf anymore (you match ncf >-< *)

The correct command would be:
```
yum update "rudder-*" "ncf*"
```

**Comment by VinceMacBuche** [comment]
PR updated

**Comment by Kegeruneku** [comment]
Accepted !!!

**Comment by VinceMacBuche** [comment]
As discussed with @Kegeruneku We decided to uniformize between zypper and yum so we now finally have:

rudder-* ncf* 

for both systems

**Pull request merged**